auth-web/src/locales/en.json

```diff
@@ -32,25 +32,25 @@
   "noPendingInvitationTitle": "No Pending Invitations",
   "noPendingInvitationMsg": "You don't have any pending invitations to BC Registries and Online Services.",
   "notifyChangeUserRoleMsg": "Notify user about this change",
-  "pendingInvitationTitle": "Thanks for joining the team {team} at BC Registries & Online Services.",
-  "pendingInvitationMsg": "The administrator for this team needs to verify your invitation before you can access this application.",
+  "pendingInvitationTitle": "Thanks for joining the account {team} at BC Registries & Online Services.",
+  "pendingInvitationMsg": "The administrator for this account needs to verify your invitation before you can access this application.",
   "noActiveUsersLabel": "No Active Users",
   "pageNotFoundTitle": "Page not found",
   "pageNotFoundMsg": "The page you are looking for has been removed or doesn't exist.",
-  "singleOwnerErrorTitle": "Unable to Leave Team",
-  "singleOwnerErrorText": "You must designate a new Owner prior to leaving the team.",
-  "confirmLeaveTeamTitle": "Confirm Leave Team",
-  "confirmLeaveTeamText": "Are you sure you wish to leave the team? If you leave, you will need an invite from an Admin to rejoin.",
+  "singleOwnerErrorTitle": "Unable to Leave account",
+  "singleOwnerErrorText": "You must designate a new Owner prior to leaving the account.",
+  "confirmLeaveTeamTitle": "Confirm Leave account",
+  "confirmLeaveTeamText": "Are you sure you wish to leave the account? If you leave, you will need an invite from an Admin to rejoin.",
   "acceptInviteLandingTitle": "Log in to accept this invitation",
-  "acceptInviteLandingMessage": "To accept the invitation to this team, log in to Cooperatives Online with your BC Services Card.",
+  "acceptInviteLandingMessage": "To accept the invitation to this account, log in to Cooperatives Online with your BC Services Card.",
   "loginBtnLabel": "Log in with BC Services Card",
   "deactivateConfirmTitle": "Deactivate your Profile",
   "deactivateConfirmText": "Deactivating your Cooperatives Online profile will remove your contact information, and your access to associated teams and/or affiliated businesses.",
   "deactivateConfirmTextEmphasis": "This action cannot be undone.",
   "deactivateFailureTitle": "Deactivation Failed",
-  "deactivateFailureText": "Unable to deactivate your profile. If you are a team owner, please dissolve your team or transfer ownership first.",
+  "deactivateFailureText": "Unable to deactivate your profile. If you are a account owner, please dissolve your team or transfer ownership first.",
```

Comment:
Needs to be changed to 'an account owner`

Reply:
done